descriptors/ModuleDescriptor-template.json

```diff
@@ -1258,7 +1258,8 @@
         "inventory-storage.service-points.item.get",
         "users.collection.get",
         "users.item.get",
-        "inventory-storage.locations.collection.get"
+        "inventory-storage.locations.collection.get",
+        "accounts.collection.get"
```

Comment:
There needs to be a dependency on the `accounts` (if that is the correct one) interface so that Okapi knows it is a dependency

Reply:
Also, this means this should not be merged until _after_ a change to our environments